models/constants.go

```diff
@@ -82,6 +82,7 @@ const (
 	PartnershipBeProf      string = "beprof"
 	PartnershipFacile      string = "facile"
 	PartnershipFpinsurance string = "fpinsurance"
+	PartnershipELeads      string = "e-leads"
```

Comment:
change into `eleads`. The signing key will be dynamically fetched from the partnership name -> `ELEADS_SIGNING_KEY`